### PR TITLE
fix(web/header) : fix the width of the export button

### PR DESCRIPTION
**Pull request by Med1233**
## Description

before the fix 
before

after the fix 
after

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x
- [ ] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/front_src/src/Header/PollerMenu/ExportConfiguration/index.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     fontSize: theme.typography.body2.fontSize,
     marginTop: theme.spacing(1),
+    width : "100%"
```
Comment (Thebarda) [inline centreon/www/front_src/src/Header/PollerMenu/ExportConfiguration/index.tsx:44]:
```suggestion
    width: '100%'
```

File: centreon/www/front_src/src/Header/PollerMenu/ExportConfiguration/index.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     fontSize: theme.typography.body2.fontSize,
     marginTop: theme.spacing(1),
+    width: '100%'
```
Comment (Thebarda) [inline centreon/www/front_src/src/Header/PollerMenu/ExportConfiguration/index.tsx:44]:
```suggestion
    width: '100%',
```

**Review by Thebarda — APPROVED** [review]
If this is a backport, you can merge it without 2 approvals

**Comment by Med1233** [comment]
> If this is a backport, you can merge it without 2 approvals

nope, this is not a backport

**Review by SoldierBoy31 — APPROVED** [review]

**Pull request merged**